docs/registry.md

```diff
@@ -0,0 +1,134 @@
+
+## List of Registry Compatibility
+
+Registries that are not listed have not been tested by the SOCI maintainers or reported by the community, but they may still be compatible SOCI.
+
+| Registry                                                                                  | Compatible? | Mechanism     | Notes                                                |
+| ----------------------------------------------------------------------------------------- | ----------- | ------------- | ---------------------------------------------------- |
+| [Docker Hub](https://hub.docker.com)                                                      | No          | N/A           | Does not support image manifests with subject fields |
+| [Amazon Elastic Container Registry (ECR)](https://aws.amazon.com/ecr/)                    | Yes         | Fallback      |                                                      |
+| [Amazon ECR Public Gallery](https://gallery.ecr.aws)                                      | No          | N/A           | Does not support image manifests with subject fields |
+| [Azure Container Registry](https://azure.microsoft.com/en-us/products/container-registry) | Yes         | Referrers API |                                                      |
+| [GitHub Packages (GHCR)](https://github.com/features/packages)                            | Yes         | Fallback      |                                                      |
+| [Google Cloud Container Registry (GCR)](https://cloud.google.com/container-registry)      | Yes         | Fallback      |                                                      |
+| [Google Cloud Artifact Registry (AR)](https://cloud.google.com/artifact-registry)         | No          | N/A           | Testing the referrers API redirects to login         |
+| [Quay.io](https://quay.io)                                                                | No          | N/A           | Does not support image manifests with subject fields |
+| [Artifactory](https://jfrog.com/artifactory/)                                             | Yes         | Fallback      |                                                      |
+| [Harbor](https://github.com/goharbor/harbor)                                              | Yes         | Fallback      |                                                      |
+| [Distribution](https://github.com/distribution/distribution)                              | Yes         | Fallback      |                                                      |
+| [OCI-playground Distribution](https://github.com/oci-playground/distribution)             | Yes         | Referrers API |                                                      |
+
+
+
+
+### Failure Examples
+
+Below are some slightly redacted examples from the services that don't support the features needed to be compatible SOCI.
+
+**Docker Hub**
+```
```

Comment:
nit: an empty line before and after the code block, also language annotation (e.g., ```shell/console/bash/etc)